### DISCUSSION TITLE
Force stop apps feature?

### DISCUSSION THREAD

**Discussion by d4rken** [d4rken-org/sdmaid-se#355] [Ideas]
Should we add a feature to AppControl that allows force stopping of apps?

"TaskKillers" are bad, so this shouldn't be a "force stop all running apps".

What would be use-cases for mass force-stopping apps?

**Comment by tathastu871** [comment]
Greenify has this festure. It clicks force stop usinh accesibility

Tipping for this feature if possible in sdmaid

**Comment by AlbiDR** [comment]
It's good to have the option but for me it was never a fundamental feature in SDM1.

From that pane it was more useful to have access to the Receiver Manager. SDM1 is basically the only app that does this (and that i trust to do it).

**Comment by Wjxfi** [comment]
Cool idea!
